bin/symfony_requirements

```diff
@@ -43,9 +43,9 @@ foreach ($symfonyRequirements->getRecommendations() as $req) {
 }
 
 if ($checkPassed) {
-    echo_block('success', 'OK', 'Your system is ready to run Symfony2 projects');
```

Comment:
@JarJak According to the specs at http://phpdoc.org/docs/latest/references/phpdoc/tags/var.html that is the recommended version and is consistent with `@param ClassName $var`.